walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,126 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public static final int DAY_START = TimeRange.START_OF_DAY;
+    public static final int DAY_END = TimeRange.END_OF_DAY;
+    public static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
+
+    /**
+      * Return a collection of TimeRange that attendees can meet for a requested 
+      * meeting. If there is not enough room for optional attendees, find TimeRanges 
```

Comment:
Instead of:

> If there is not enough room for optional attendees, [...]

it might be better to say something like 

> If there are no TimeRanges that work for both required and optional attendees, [...]

Reply:
The comment is updated! :)